### PR TITLE
Add Mixware Hyper-S printer definition to Ultimaker Cura

### PR DESCRIPTION
**Pull request by mixware011**
Hello Team Cura,

We've developed a new FDM 3D printer, named Mixware Hyper-S.

We wants to add our printer profile definition i.e. Mixware Hyper-S in ultimaker Cura slicer.

### CODE REVIEW

**Comment by nallath** [comment]
The STL you provided is empty.

**Review by nallath — CHANGES_REQUESTED** [review]
I've looked at a random sample of your settings and pretty much all of them are re-defined in some form or another. 

It will make it a lot easier to maintain if you don't re-define settings like that. 

Please go over the provided settings once more and check if it's actually needed to change them.

File: resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[metadata]
+setting_version = 16
+type = quality
+quality_type = normal
+global_quality = True
+
+[values]
+layer_height = 0.1
+infill_before_walls = False
+acceleration_enabled = True
```
[the remark this answers is not in the document]
Comment (mixware011) [reply resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg]: I hadn't thought about easier maintenance.
Now I resubmit.

File: resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = normal
+global_quality = True
+
+[values]
+layer_height = 0.1
+infill_before_walls = False
```
[the remark this answers is not in the document]
Comment (mixware011) [reply resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg]: I hadn't thought about easier maintenance.
Now I resubmit.

File: resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+acceleration_enabled = True
+adhesion_type = raft
+jerk_enabled = True
```
[the remark this answers is not in the document]
Comment (mixware011) [reply resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg]: I hadn't thought about easier maintenance.
Now I resubmit.

File: resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+support_enable = True
```
[the remark this answers is not in the document]
Comment (mixware011) [reply resources/quality/mixware_hyper_s/mixware_hyper_s_normal.inst.cfg]: I hadn't thought about easier maintenance.
Now I resubmit.

**Review by konskarm — CHANGES_REQUESTED** [review]

**Review by Ghostkeeper — APPROVED** [review]
Aside from that there were apparently different setting values previously, the current setting values look good to me.

This PR kind of got lost due to not having taken off the "needs info" label, because the issue with the platform adhesion and jerk was never addressed. But you seem to be maintaining it still and so we'll assume that it prints correctly.

I'll update the `setting_version`, give these profiles a test and we can then maybe finally merge them into mainline Cura.

**Comment by Ghostkeeper** [comment]
I fixed up the tests that were failing. No idea why Github thinks there were 2000 commits in this PR though. There were only 50. It's got to do with the push I did to the upstream repository I think, but that push contains only commits from mainline Cura.

One point of concern is that you probably intended this to come along with some Mixware material profiles, but those profiles are not submitted to the fdm_materials repository. I changed it to default to generic_pla instead.

**Pull request merged**